codegen.ts

```diff
@@ -0,0 +1,17 @@
+import type { CodegenConfig } from '@graphql-codegen/cli'
+
+const config: CodegenConfig = {
+  overwrite: true,
+  schema: './src/graphql/data/schema.graphql',
+  documents: ['./src', '!**/node_modules/**', '!**/__mocks__/**', '!**/__generated__/**', '!**/thegraph/**'],
```

Comment:
nit. i think it's cleaner to enforce queries to live in `ts` or `graphql` types explicitly (on mobile right now we enforce queries to all be in the same file, although i expect we will split it up post-migration   `documents: 'src/data/queries.graphql'`)